backend/server.js

```diff
@@ -0,0 +1,20 @@
+const restify = require("restify");
+const axios = require("axios");
+const corsMiddleware = require("restify-cors-middleware");
+
+const server = restify.createServer();
+const cors = corsMiddleware({
+  origins: ["*"]
+});
+server.use(cors.actual);
+server.get("/quorum", async (req, res, next) => {
+  const response = await axios.get(
+    "http://0.0.0.0:11626/quorum?transitive=true"
```

Comment:
For deployment purposes, it might make sense to also make this server URL configurable via env var.